tests/make_observations.py

```diff
@@ -13,6 +13,8 @@
 TEST_OBSERVATION_FILE = os.path.join(
     os.path.dirname(__file__), "data", "observations.h5"
 )
+RANDOM_SEED = 2023
+np.random.seed(RANDOM_SEED)
```

Comment:
To be pedantic for a second:

This is a little dodgy because it modifies `np.random` for _all_ tests within the same Python interpreter run. So other tests will also get your seed - or you might get theirs. For almost all practical purposes, that should be fine, but if we had any "golden" tests (that is, tests that a particular output was unchanged across versions, like "gold plates") which involved randomness, this could break them in a very hard-to-expect way. 

An alternative strategy which wouldn't have this problem would be to make a new Numpy `Generator` within your test:

```python
rng = np.random.default_rng(seed=2023)
offset_from_start = rng.uniform(dts, dts + exposure_duration / 86400)
```

This would insulate things. Almost definitely not necessary, but hey, why review code if you don't get to pass in pedantic commentary like this?

Reply:
Pedantisms appreciated and incorporated. A much cleaner change and I've learned about a new `numpy` feature.